mandatory/3-playing-computer.js

```diff
@@ -9,10 +9,10 @@
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  4. How many times is "f1" called? 2 time
+  5. How many times is "f2" called? 2 time
+  6. What value does the "a" parameter take in the first "f1" call? 7
```

Comment:
This isn't a very well worded question, it  confused me at first too! I think it's asking for what value is put into the function i.e. looking at function f2:

const f1 = function (a, b) {
  return a + b;
};

it wants to know what value is being put in the a parameter. Like I said, it's not the easiest question and wasn't written very well I don't think

Reply:
you know when i ==1 we can call f1 so its 7
im not sure

Reply:
Hey, yes so for this question the answer is 1 because i == 1 and acts as the parameter where a is in f1